### ISSUE TITLE
Caster Conduct

### ISSUE THREAD

**Issue by synthparadox** [chairleague/hots-rules#5]
I suggest putting in a section codifying caster conduct. Such as casters are responsible for showing up on time and if there is a caster assigned to your match it is their responsibility to host the lobby/mark results.
Also perhaps dictating a chain of command in case there are issues.

**Comment by johnafogarty4** [comment]
Yes please, this.  If we are all doing it the same way it makes it easier on the player and the caster.  I can take a stab at it, if everyone agrees this is a good idea.
